rules/windows/process_creation/proc_creation_win_event_logging_disable_via_key_minint.yml

```diff
@@ -0,0 +1,46 @@
+title: Security Event Logging Disabled Via MiniNt Registry Key - Process
+id: 1a4bd6af-99ac-4466-b5b2-7b72b4a05462
+related:
+    - id: 8839e550-52d7-4958-9f2f-e13c1e736838 # Disable Security Events Logging Adding Reg Key MiniNt - Registry Set
+      type: similar
+status: experimental
+description: |
+    Detects attempts to disable security event logging by adding MiniNt registry key.
+    This key is used to disable the Windows Event Log service, which collects and stores event logs from the operating system and applications.
+    Adversaries may want to disable this service to prevent logging of security events that could be used to detect their activities.
```

Comment:
```suggestion
    Detects attempts to disable security event logging by adding the `MiniNt` registry key.
    This key is used to disable the Windows Event Log service, which collects and stores event logs from the operating system and applications.
    Adversaries may want to disable this service to prevent logging of security events that could be used to detect their activities.
```